CONTRIBUTING.md

```diff
@@ -130,19 +130,19 @@ You are almost ready to make changes to files, but before that you should **alwa
 
     You should get an output like this:
     
-        On branch master
-        Your branch is up-to-date with 'origin/master'.
+        On branch main
+        Your branch is up-to-date with 'origin/main'.
         nothing to commit, working directory clean
 
-    If you are not on master or your working directory is not clean, resolve any outstanding files/commits and checkout _master_:
+    If you are not on main or your working directory is not clean, resolve any outstanding files/commits and checkout _master_:
 
     ```sh
-    git checkout master
+    git checkout main
     ```
 
 2. Sync the latest changes from the upstream **Chapter** _master_ branch to your local fork's _master_ branch. This is very important to keep things synchronized and avoid "merge conflicts".
```

Comment:
@ojeytonwilliams it looks like there are at least two more references to master on this line.

Did you find and replace on master or some other pattern?  Could the references to underscores have been missed `_master_` ?

Reply:
Yeah, I did - I must have had it set to match whole words. I'll fix that in a sec.